modules/layers/src/path-layer/path-layer.js

```diff
@@ -311,7 +316,7 @@ export default class PathLayer extends Layer {
 
     let i = 0;
     paths.forEach(path => {
-      for (let ptIndex = 1; ptIndex < path.length; ptIndex++) {
+      for (let ptIndex = 1; ptIndex < this._getPathLength(path); ptIndex++) {
```

Comment:
Call it once and assign to constant

Reply:
> Call it once and assign to constant

Restored the original code in this function since it cannot be generalized simply by reporting the right path length. I made the change by mistake here.

All the functions below have been updated accordingly.